plugins/woocommerce-blocks/assets/js/base/components/drawer/style.scss

```diff
@@ -140,9 +145,6 @@ $drawer-animation-duration: 0.3s;
 		outline: none;
 	}
 
-	> span {
```

Comment:
We disable the text wrapper in this button via `removeTextWrap` which I introduced, so this is not needed now.